### DISCUSSION TITLE
Expand data from api

### DISCUSSION THREAD

**Discussion by josecao** [KevinVandy/material-react-table#134] [General]
Hi author,

I want to expand one row and data of this row , it was get from api.  
I find it in https://www.material-react-table.dev/?path=/docs/features-sub-row-tree-examples--sub-row-tree-enabled-default

**Comment by KevinVandy** [comment]
There is no example for lazy loading expanded child rows in the docs yet, but maybe this conversation will help give an idea for how to approach the problem https://github.com/TanStack/table/issues/1884
